### DISCUSSION TITLE
Mark un-translated texts in language.h

### DISCUSSION THREAD

**Discussion by andimik** [PE5PVB/TEF6686_ESP32#371] [Ideas]
In order to maintain the language file and hopefully get all translations soon I propose to add a comment if a phrase has not been translated so far.

For example

```
    "Wi-Fi active", // 51
```

gets 

```
    "Wi-Fi active", // 51 still to be translated
```

**Comment by PE5PVB** [comment]
Good idea. You know how it works ;-)

**Reply by andimik** [reply]
I know this feature from Qt-Linguist :-)

**Comment by PE5PVB** [comment]
Then, I friendly invite you to do it ;-)

**Comment by andimik** [comment]
Sure, already started it, but will continue tomorrow.

**Reply by PE5PVB** [reply]
Good boy

**Comment by andimik** [comment]
thanks for merging https://github.com/PE5PVB/TEF6686_ESP32/pull/375/commits/43aeec73fdcd0e6bc00aa8e0bf0d03c6d1ac2e4b
